### DISCUSSION TITLE
Run Carla along with NVIDIA Omniverse

### DISCUSSION THREAD

**Discussion by kshitiz-deloitte** [carla-simulator/carla#6631] [Q&A]
Hi Everyone, 
I want to use Carla to generate synthetic data along with omniverse replicator.

So is it possible to connect Carla with Omniverse replicator?
If yes, can you provide some kind of guidance or documentation regarding the same.

I asked this question because of following link:
[CARLA Autonomous Driving Consortium Names NVIDIA as Latest Member] (http://carla.org/2022/09/20/nvidia-joins-carla-consortium/)

> Connecting the CARLA simulator to [NVIDIA Omniverse] (https://www.nvidia.com/en-us/omniverse/) , a platform for integrating and building custom 3D pipelines, will provide rich new sources of content such as vehicles, pedestrians, and props for simulation in CARLA.

**Comment by NirobNabil** [comment]
Hello, I'm also trying to achieve a similar objective. did you find any success?
